### ISSUE TITLE
Support adding/removing mutliple repositories in the "InstallationRepositories::added::removed" event

### ISSUE THREAD

**Issue by begedin** [code-corps/code-corps-api#839]
# Problem

https://developer.github.com/v3/activity/events/types/#installationrepositoriesevent

```JSON
"repositories_added": [

],
"repositories_removed": [
  {
    "id": 1296269,
    "name": "Hello-World",
    "full_name": "octocat/Hello-World"
  }
],
```

The above keys in the payload are arrays/lists, meaning they can contain multiple records. In the implementation we currently have in #834, we assume there's just one item in there. We should not assume so.

Not handling this and assuming the implementation in #834 gets merged, our event will get into the "errored" state.

# Solution

- try and find the `GithubAppInstallation` first
- once that's done, try and find the `GithubRepo` for each of the attributes. depending on it existing, and the action being performed, add or delete it. 
- in case of deletion, make sure associated `ProjectGithubRepo` records are also deleted
- skip creating records which already exist, ignore deleting records which do not exist

**Comment by begedin** [comment]
This is now actually done as part of #834

**Comment by joshsmith** [comment]
Closed by #834.
